content/blog/2016/branding-type.md

```diff
@@ -170,15 +176,27 @@ method of mixing and matching, but neither did we feel that any one
 element collage encapsulated our brand. We tried style tiles, but those
 felt both too specific and not specific enough.
 
-![image][1]
+{% call embed.img(
+    src='blog/2016/brand/type-styletiles.jpg',
+    alt=''
+) %}
+{% endcall %}
 
 *Style Tiles - 4 cropped sections*
 
-![image][2]
+{% call embed.img(
+    src='blog/2016/brand/type-elcollage.jpg',
+    alt=''
+) %}
+{% endcall %}
 
 *Element Collage*
```

Comment:
We could turn these into actual `figure`s with `figcaption` - maybe I'll make a new story for general embed cleanup.